modules/katamari/src/test/ts/atomic/api/data/ResultErrorTest.ts

```diff
@@ -1,147 +1,147 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
-import { Testable } from '@ephox/dispute';
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
 import fc from 'fast-check';
 import * as Fun from 'ephox/katamari/api/Fun';
-import { Optional } from 'ephox/katamari/api/Optional';
-import { tOptional } from 'ephox/katamari/api/OptionalInstances';
 import { Result } from 'ephox/katamari/api/Result';
-import { tResult } from 'ephox/katamari/api/ResultInstances';
 import { arbResultError, arbResultValue } from 'ephox/katamari/test/arb/ArbDataTypes';
+import { assertNone } from 'ephox/katamari/test/AssertOptional';
+import { assertResult } from 'ephox/katamari/test/AssertResult';
+
+describe('atomic.katamari.api.data.ResultErrorTest', () => {
+
+  it('unit tests', () => {
+    const s = Result.error('error');
+    assert.deepEqual(s.is('error'), false);
+    assert.deepEqual(s.isValue(), false);
+    assert.deepEqual(s.isError(), true);
+    assert.deepEqual(s.getOr(6), 6);
+    assert.deepEqual(s.getOrThunk(() => 6), 6);
```

Comment:
```suggestion
    assert.isFalse(s.is('error'));
    assert.isFalse(s.isValue());
    assert.isTrue(s.isError());
    assert.equal(s.getOr(6), 6);
    assert.equal(s.getOrThunk(() => 6), 6);
```
These are all primitive values, so we shouldn't need to use `deepEqual` here I believe.